### DISCUSSION TITLE
What is the current state of affairs for PWM servos on a raspberry pi 5?

### DISCUSSION THREAD

**Discussion by Brunnian** [gpiozero/gpiozero#1128] [Q&A]
I find using the _AngularServo_ method from gpiozero that the servo moves through half the expected angle and jitters once it gets there.  There is actually less jitter on e.g. gpio 24 than on gpio 12, despite gpio 12 being a dedicated PWM pin.

I get the message below when running

> /usr/lib/python3/dist-packages/gpiozero/output_devices.py:1509: PWMSoftwareFallback: To reduce servo jitter, use the pigpio pin factory.See https://gpiozero.readthedocs.io/en/stable/api_output.html#servo for more info

But it appears that pigpio is not available for the raspberry pi 5. 

Does anyone know of a solution to reliable control of servos?

**Comment by bennuttall** [comment]
The equivalent of pigpio for Pi 5 is rgpio (not to be confused with rpi.gpio) - which we don't yet have support for.

**Reply by Brunnian** [reply]
Thanks for the confirmation, at least.  Is there a plan, or a timescale?
